src/python/CMakeLists.txt

```diff
@@ -58,36 +69,51 @@ if (APPLE)
 #    set (PYTHON_LIBRARIES /opt/local/lib)
 endif ()
 
+# Use python executable to find the site packages dir if not defined
```

Comment:
I've been debating this code or just having  default value of:
${CMAKE_INSTALL_LIBDIR}/python${PYTHON_VERSION_FOUND}/site-packages

But I think some distros use python3/site-packages? Then some don't even call it site-packages. I was trying to be generic here, but I just don't know how I feel about this bit of code. What do you all think?

Reply:
Apparently this is what automake does.

Reply:
You mean that automake uses the python executable to find the site packages dir? This strikes me as the most foolproof thing, no?

Reply:
Yeah. I was told that automake does something very similar to this to find the site packages dir. I think there is a way to make it more robust, but I need to as Soap about the idea.